pkg/aws/helpers.go

```diff
@@ -352,11 +354,30 @@ func GetOperatorPolicyARN(accountID string, prefix string, namespace string, nam
 }
 
 func GetPolicyARN(accountID string, name string) string {
-	return fmt.Sprintf("arn:aws:iam::%s:policy/%s", accountID, name)
+	partition := "aws"
+	region, err := GetRegion(arguments.GetRegion())
```

Comment:
Given an AWS region you could also use `endpoints.PartitionForRegion` https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#PartitionForRegion so that this wouldn't have to be hardcoded

something like:
```go
partition, ok := endpoints.PartitionForRegion(endpoints.DefaultPartitions(), region)
if !ok {
	 return nil, fmt.Errorf("no AWS partition found for region %s", region)
}
```

Reply:
Nice!! I was wondering if there was something like this, I was looking in the `arn` pkg and couldn't see anything.